### DISCUSSION TITLE
HELP！ERROR is : javax.ws.rs.core.NoContentException

### DISCUSSION THREAD

**Discussion by houxd98** [Netflix/conductor#3347] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49725014/202176549-12ededa5-aa8c-4e5e-8b7e-567e463e5030.png)
 I'm a new learner of conductor. 
I write a worker by java. 
But when I run it,  the above error message appears in the dashboard.
Has anyone ever met  this situation? What should I do?

**Comment by v1r3n** [comment, marked as answer]
Hi, there is a PR out to fix this, in the meantime, you can update the gradle dependencies to include the requirements as follows:

https://github.com/Netflix/conductor/pull/3320/files#diff-2019b353b44fd4986682eb4efe5d3e99ce12fdd7ba343f4f581fef47f88cdc38

```
implementation "javax.ws.rs:javax.ws.rs-api:${revJAXRS}"
implementation "org.glassfish.jersey.core:jersey-common:${revJerseyCommon}"
```

**Reply by houxd98** [reply]
Thanks for your reply, it's very helpful.
